content/id/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -0,0 +1,459 @@
+---
+reviewers:
+- phanama
+- wahyuoi
+title: Horizontal Pod Autoscaler
+feature:
+  title: Horizontal scaling
+  description: >
+    Scale your application up and down with a simple command, with a UI, or automatically based on CPU usage.
+
+content_template: templates/concept
+weight: 90
+---
+
+{{% capture overview %}}
+
+Horizontal Pod Autoscaler secara otomatis akan men-*scale* jumlah *pod* didalam kontroler replikasi, *deployment*, 
```

Comment:
Horizontal Pod Autoscaler, pod, replication controller, deployment, replicaset, stateful are all kubernetes API objects, should be written in CamelCase format.
```suggestion
HorizontalPodAutoscaler secara otomatis akan memperbanyak (_scale_) jumlah Pod di dalam ReplicationController, Deployment, 
```

Reply:
Changed, please check